arangod/Cluster/AgencyCache.cpp

```diff
@@ -469,3 +567,129 @@ void AgencyCache::invokeAllCallbacks() const {
   }
   invokeCallbacks(toCall);
 }
+
+
+AgencyCache::change_set_t AgencyCache::changedSince(
+  std::string const& what, consensus::index_t const& last) const {
+
+  static std::vector<std::string> const planGoodies ({
```

Comment:
would be nice to move local static variables outside function context to speedup access

Reply:
also tested just now with google benchmark not measureable difference in runtime. in that case would rather not make the change because we lack the time to run complete chaos for over a week again.